04.ls/ls.rb

```diff
@@ -47,39 +53,35 @@ def output(file_names, directory_path, options)
 end
 
 def output_long_listing_format(file_names, directory_path)
-  puts "total #{calc_block_count_total(file_names, directory_path)}"
-
   details_by_file_name = build_details_by_file_name(file_names, directory_path)
 
-  max_width_by_detail = calc_max_width_by_detail(details_by_file_name)
+  puts "total #{calc_block_count_total(details_by_file_name)}"
```

Comment:
```suggestion
  total = details_by_file_name.sum { |h| h[:blocks] }
  puts "total #{total}"
```

とかの方がスッキリしてわかりやすいかも
https://bootcamp.fjord.jp/pages/when-you-should-define-method